.travis.yml

```diff
@@ -1,14 +1,6 @@
 language: java
 jdk:
     - oraclejdk8
-notifications:
```

Comment:
For PR purposes I would split this into a separate Dwolla-customizations commit

Reply:
Done.